api/app/serializers/spree/v2/storefront/payment_serializer.rb

```diff
@@ -0,0 +1,14 @@
+module Spree
+  module V2
+    module Storefront
+      class PaymentSerializer < BaseSerializer
+        set_type :payment
+
+        attributes :amount, :state, :response_code, :avs_code, :number, :cvv_response_code,
```

Comment:
let's expose only `amount`, `response_code , `number`, `cvv_response_code` and `cvv_response_message`